### ISSUE TITLE
Application fatal error

### ISSUE THREAD

**Issue by Invisible04** [Sketchware-Pro/Sketchware-Pro#356]
App crash after open

**Comment by JavkhlanK** [comment]
Need more details. Are you talking about Sketchware Pro or your project made with Sketchware Pro? Which Sketchware Pro version did you use, and anything more about your issue are a must before we can help you.

**Comment by Arashvscode** [comment]
> App crash after open

First enter the settings and submit the required permissions, then the program will run well lol

**Comment by khaled-0** [comment]
Fixed with rc2

**Comment by JavkhlanK** [comment]
Since there hasn't been any response I guess it's a duplicate of #298.
